### DISCUSSION TITLE
Events with multiple time slots

### DISCUSSION THREAD

**Discussion by rich-howell** [pretix/pretix#3231] [Q&A]
Hello,

I have an event lets call it “Summer Special” it runs on July 7th but on that day there are 7 time slots available 10:30, 11:00, 12:15, 13:30. 14:45. 16:00 and 16:55 on the Pretix calendar each of these time slots show up as a single entry for the customer to click. 

Is it possible to create an event on the calendar called “Summer Special” then when the customer clicks the event the available dates are shown rather than clog the calendar?

**Comment by raphaelm** [comment]
No, that's not possible, the only thing that would come close would be using the "list" version of the event list.
